### PR TITLE
Document modular inclusion

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Added documentation for cherry picking parts of the framework.

## QA
Check code is ok

## Details
Fixes https://github.com/vanilla-framework/vanilla-framework/issues/412

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,26 @@ $color-brand: #ffffff;
 // Add theme if applicable
 ```
 
-If you don't want the whole framework, you can just `@include` specific [parts](scss) - e.g. `@include vf-b-forms`.
+### Advanaced usage: Modular inclusion
```
Comment (barrymcgee) [inline README.md]: spelling - "Advanaced"

**Comment by barrymcgee** [comment]
I'm not sure this cherry-pick approach from any part of the framrowrk will work as it's only really the patterns part of the stack that is fully encapsulated. 

All pattern styling is based on the reset already being applied not to mention other base styles, you can't just knock out lower layers and expect the patterns to still stand up.

**Comment by barrymcgee** [comment]
For this to work, we'd need a catch-all mixin for all base styles at least and then allow the user to cherry pick whichever patterns they need?

**Comment by anthonydillon** [comment]
@barrymcgee your right. Closing this PR and passed your comment back up to the issue.

**Pull request closed without merging**